boards/cats_rev1Pro/src/cli/cli.c

```diff
@@ -873,6 +1026,17 @@ static void processCharacterInteractive(const char c) {
       cliBuffer[--bufferIndex] = 0;
       cliPrint("\010 \010");
     }
+  } else if (c == 27) {
```

Comment:
we should have defines for these ascii values at some point, don't have to change it now

Reply:
Yeah this is weird, the up arrow does not behave as a single character and I just didn't bother to look at multiple bytes.
27 is the first thing that is sent when pressing any arrow button, so you can also use the left and right keys to get the last entry for example.